logic/src/test/nl/jft/logic/statistic/impl/TitleStatisticTest.java

```diff
@@ -0,0 +1,8 @@
+package nl.jft.logic.statistic.impl;
+
+/**
+ * @author Lesley
+ */
+public class TitleStatisticTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS2187)